legacyman_parser/dita/create_regions.py

```diff
@@ -0,0 +1,62 @@
+from xml.dom import minidom
```

Comment:
could you move this code into `legacy_publisher\dita_publisher`, since itś related to publishing dita data, rather than parsing data